application.js

```diff
@@ -11,9 +11,6 @@ securableEndpoints = [];
 
 var app = express();
 
-// Enable CORS for all requests
-app.use(cors());
```

Comment:
I had to remove any use of cors/bodyParser middlewares since they prevents simple request proxying in `/try` route.